src/ShapeCrawler/Drawing/IShapeFill.cs

```diff
@@ -47,4 +47,9 @@ public interface IShapeFill
     ///     Fills the shape with solid color in hexadecimal representation.
     /// </summary>
     void SetColor(string hex);
+
+    /// <summary>
+    ///     Removes Fills from the shape. And set a A.NoFill element.
```

Comment:
What about just "Removes Fill"?

Reply:
No problem.

Your idea is for the users to not have to know or care about underlying OXML right?

Reply:
Right. The eventual goal is to have an user friendly Object Model that covers the whole functionalities without having to think about underlying OOXML.